### PR TITLE
Update wp-cli to the latest version

### PR DESCRIPTION
**Pull request by tmatsuo**
Seems like we need to update `wp-cli` for newer WordPress.

### CODE REVIEW

**Comment by tmatsuo** [comment]
Seems like Travis test for php-nightly is failing. Can we ignore tests for php-nightly?

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,8 @@ matrix:
     - php: 5.6
     - php: hhvm
     - php: nightly
+  allow_failures:
+    - php: nightly
```
Comment (bshaffer) [inline .travis.yml]: I do not think we should do this...

Comment (bshaffer) [inline .travis.yml]: Oh sorry, I thought this was for **our** nightly builds. We should add PHP 7 and remove nightly.

Comment (tmatsuo) [inline .travis.yml]: Ah good point. Done

**Comment by tmatsuo** [comment]
@bshaffer PTAL

**Comment by bshaffer** [comment]
LGTM!

**Pull request merged**